app/generate.js

```diff
@@ -97,8 +99,24 @@ function generatePages(renderedComponents) {
       page.conclusionHtml = '';
     }
 
-    page.header = header;
-    page.footer = footer;
+    if (!page.header) {
+      page.header = defaultHeader;
+    } else {
+      if (!page.header.logo) {
+        throw new Error('Header must have logo');
+      }
+      page.header = headerTemplate({
```

Comment:
I think the output of `headerTemplate` needs to land in `page.header.html` for the code to work. Right?

Reply:
Check updated pullrequest